terraform/deploy/app/modules.tf

```diff
@@ -43,4 +45,10 @@ module "emr" {
     id      = data.terraform_remote_state.management.outputs.artefact_bucket.id
     kms_arn = data.terraform_remote_state.management.outputs.artefact_bucket.cmk_arn
   }
+  region      = var.region
+  account     = local.account[local.environment]
+  environment = local.environment
+
+  truststore_certs   = "s3://${data.terraform_remote_state.certificate_authority.outputs.public_cert_bucket.id}/ca_certificates/dataworks/dataworks_root_ca.pem,s3://${data.terraform_remote_state.mgmt_ca.outputs.public_cert_bucket.id}/ca_certificates/dataworks/dataworks_root_ca.pem,s3://${data.terraform_remote_state.certificate_authority.outputs.public_cert_bucket.id}/ca_certificates/dataworks/ca.pem,s3://${data.terraform_remote_state.mgmt_ca.outputs.public_cert_bucket.id}/ca_certificates/dataworks/ca.pem"
+  truststore_aliases = "dataworks_root_ca,dataworks_mgt_root_ca,env_ca,mgt_ca"
```

Comment:
You don't actually need env_ca or mgt_ca, since DKS is already using the new CA. Please remove these.

Reply:
Removed.